admin/nodes/serializers.py

```diff
@@ -24,7 +24,7 @@ def serialize_node(node):
         'children': map(serialize_simple_node, node.nodes),
         'deleted': node.is_deleted,
         'pending_registration': node.is_pending_registration,
-        'creator': node.creator._id,
+        'creator': node.creator._id if node.creator else None,
```

Comment:
You're right! I added this cause I had some bad data in my DB I was trying to account for. Will fix :star: